actors/builtin/miner/miner_test.go

```diff
@@ -877,7 +877,7 @@ func TestCommitments(t *testing.T) {
 
 			// Too big at version 4
 			proveCommit := makeProveCommit(sectorNo)
-			proveCommit.Proof = make([]byte, 1920)
+			proveCommit.Proof = make([]byte, 192)
```

Comment:
Not sure if its important to keep this case but this proof size is not too big for version 4 which had [a limit of 1024](https://github.com/filecoin-project/specs-actors/commit/c5e31f2282adc206030f5fbff1af0814c0765535#diff-25d448ee2846a665efe3de55eab372d83c4a4ef4b27108a0250b03439c6b48f0R69) without taking into account seal proof type.

Reply:
I'll try to improve the test (i.e., test with all sector sizes).